src/components/Book.js

```diff
@@ -46,19 +44,21 @@ const Book = (props) => {
               </div>
             </div>
             <div className="percent">
-              <span className="circle" style={progress}> </span>
-              <div>
+              <span className="circle" style={progress}>
+                <div className="inner"> </div>
+              </span>
+              <div className="progress-contain">
                 <h3>
-                  {percentComplete}
+                  {prog}
                   %
                 </h3>
                 <p>Completed</p>
               </div>
             </div>
-
-            <div>
-              <h4>Current Chapter</h4>
-              <h2>{chapter}</h2>
+            <div className="border"> </div>
+            <div className="current-div">
+              <h4 className="current">Current Chapter</h4>
+              <h2 className="chapter">Chapter 17</h2>
```

Comment:
- [ ] Lets ensure there is some padding to the right of the book.Check the design below for comparison:
template's
![image](https://user-images.githubusercontent.com/68167810/132091988-c9061e91-bdaf-4eb2-a401-bfa70d4d29ee.png)
==========================================================================
your's
![image](https://user-images.githubusercontent.com/68167810/132092009-4c4c3de0-33bc-4fc5-8b34-278f8defc9ee.png)